DDCore/src/DetectorImp.cpp

```diff
@@ -164,15 +164,24 @@ DetectorImp::DetectorImp()
 DetectorImp::DetectorImp(const string& name)
   : TNamed(), DetectorData(), DetectorLoad(this), m_buildType(BUILD_NONE)
 {
+#if defined(DD4HEP_USE_GEANT4_UNITS) && ROOT_VERSION_CODE > ROOT_VERSION(6,22,7)
```

Comment:
```suggestion
#if defined(DD4HEP_USE_GEANT4_UNITS) && ROOT_VERSION_CODE >= ROOT_VERSION(6,22,7)
```
?

Reply:
Welche versio jetzt zum Kuckuck?

Reply:
Hier war die frage ob groesser gleich, so wie im naechsten elif

Reply:
groesser oder groesser gleich ist wurscht solange die logic stimmt.
Wenns ab 6.22.08 drin ist, dann > 6.22.07 oder >= 6.22.08 wie oben gesagt
und NICHT >= 6.22.07

Reply:
Soll denn hier
https://github.com/AIDASoft/DD4hep/blob/23fc86e951a85a53be8d0247750a96cf2494663c/DDCore/src/DetectorImp.cpp#L174
und hier
https://github.com/AIDASoft/DD4hep/blob/23fc86e951a85a53be8d0247750a96cf2494663c/DDCore/src/DetectorImp.cpp#L167
Nicht das gleiche geprueft werden?

Ausserdem ist im HEAD vom 6.22 patches branch
https://github.com/root-project/root/blob/ea52077681a8cd9158ea717e83f0f1fd76af2a27/core/base/inc/RVersion.h#L21
```
#define ROOT_VERSION_CODE ROOT_VERSION(6,22,7) /* 398855 */
```
wo deine Aenderungen auch drin sind, also muss es `>= 6.22.07 ` sein, oder nicht?

Reply:
Das wird langsam kompliziert....Das heisst man haengt dann davon ab wann 6,22 patches ausgecheckt wurde..

Reply:
Entweder jemand aendert ROOT so dass es backward compatible ist, oder wir muessen hier auf `>=` aendern und mergen (weil sonst die LCG vies in cvmfs nicht updaten), oder?